tests/robottelo/conftest.py

```diff
@@ -0,0 +1,45 @@
+import glob
+import os
+from pathlib import Path
+from tempfile import NamedTemporaryFile
+
+import pytest
+from fauxfactory import gen_string
+
+
+@pytest.fixture(scope='function')
```

Comment:
You can drop the explicit scope of you want

Reply:
But I put it there just for you! And because the other fixture already had it set explicitly.

Reply:
Consistency is a wonderful thing